code/frameworks/sveltekit/src/portable-stories.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  composeConfigs,
+  setProjectAnnotations as originalSetProjectAnnotations,
+  setDefaultProjectAnnotations,
+} from 'storybook/internal/preview-api';
+import type {
+  NamedOrDefaultProjectAnnotations,
+  NormalizedProjectAnnotations,
+  ProjectAnnotations,
+} from 'storybook/internal/types';
+
+import type { SvelteRenderer } from '@storybook/svelte';
+
+// TODO: investigate whether we need to prebundle this or can just import from the renderer package
+import { INTERNAL_DEFAULT_PROJECT_ANNOTATIONS as svelteAnnotations } from '../../../renderers/svelte/src/portable-stories';
```

Comment:
Why not import `@storybook/svelte` instead?

Reply:
oh that is the TODO :P

Reply:
That's why I added a TODO there, I think there will be issues with a .svelte file which is imported as part of `'@storybook/svelte/internal/PreviewRender.svelte'`, we need to investigate!

Reply:
This is fixed!